screens/EinvoiceStatusScreen.js

```diff
@@ -176,7 +177,7 @@ class EinvoiceStatusScreen extends React.Component {
                                 </StyledText>
                             </View>
                             <View style={{flex: 2}}>
-                                <StyledText>{formatDate(item.createdTime)}
+                                <StyledText>{formatDate(item.createdTime, TimeZoneService.getTimeZone())}
```

Comment:
Is there a need to pass timezone here after you refactored formatDate function?

Reply:
miss, remove